pallets/creditcoin/src/mock.rs

```diff
@@ -294,3 +295,14 @@ pub fn pending_rpc_request(
 		..Default::default()
 	}
 }
+
+#[test]
+#[tracing_test::traced_test]
+fn offchain_worker_should_log_when_authority_is_missing() {
+	ExtBuilder::default().build_offchain_and_execute(|| {
+		System::set_block_number(1);
+
+		Creditcoin::offchain_worker(System::block_number());
+		assert!(logs_contain("Not authority, skipping off chain work"));
+	});
+}
```

Comment:
Strangely enough there's no change in reported coverage for the relevant line:
https://app.codecov.io/gh/gluwa/creditcoin/blob/5a615a4e4df14454f36c536bd963030910ecb315/pallets/creditcoin/src/lib.rs

@nathanwhit any ideas why ?